app/services/db/identity/sp_user_counts.rb

```diff
@@ -13,7 +13,38 @@ def self.call
           WHERE identities.service_provider = service_providers.issuer
           GROUP BY identities.service_provider ORDER BY identities.service_provider
         SQL
-        ActiveRecord::Base.connection.execute(sql)
+        ActiveRecord::Base.connection.execute(sql).to_a
+      end
+
+      def self.overall
+        sql = <<~SQL
+          SELECT
+            COUNT(*) AS num_users
+          , by_user.is_ial2
+          FROM (
+            SELECT
+              identities.user_id
+            , BOOL_OR(identities.verified_at IS NOT NULL) AS is_ial2
+            FROM identities
+            GROUP BY identities.user_id
+          ) by_user
+          GROUP BY by_user.is_ial2
+        SQL
+
+        results = ActiveRecord::Base.connection.execute(sql).to_a
+
+        ial1_total = results.find { |r| !r['is_ial2'] }&.fetch('num_users') || 0
+        ial2_total = results.find { |r| r['is_ial2'] }&.fetch('num_users') || 0
+
+        [
+          {
+            issuer: 'LOGIN_ALL',
```

Comment:
would using `nil` be valid here? It's unlikely, but this value could be an issuer that exists

Reply:
I'll run it by Chris, I figured having a value like this would be easier to read, but you make a good point

Reply:
Confirmed it's good to use `nil`, updated in 0305d375b69f609bf94f6ab12846039faa85f656. thanks!